ymap/operators.py

```diff
@@ -37,7 +39,13 @@ def poll(cls, context):
 
     def run(self, context):
         ymap_obj = context.active_object
-        create_ymap_group(sollum_type=SollumType.YMAP_ENTITY_GROUP, selected_ymap=ymap_obj, empty_name='Entities')
+        ymap_group = create_ymap_group(sollum_type=SollumType.YMAP_ENTITY_GROUP, selected_ymap=ymap_obj, empty_name='Entities')
+        if ymap_group:
+            bpy.ops.object.select_all(action='DESELECT')
+            ymap_group.select_set(True)
+            context.view_layer.objects.active = ymap_group
+            # TODO: Find a way to use "bpy.ops.outliner.show_active()" to show the new object in outliner. But we are in wrong context here.
```

Comment:
`context.temp_override()` allows changing the context. Something like this works when run as a script, should work here as well:
```py
for area in context.screen.areas:
    if area.type != "OUTLINER":
        continue

    for region in area.regions:
        with context.temp_override(area=area, region=region):
            bpy.ops.outliner.show_active()
```

Reply:
Unfortunately I can't manage to get it to work